tests/rptest/services/redpanda_cloud.py

```diff
@@ -1562,3 +1567,49 @@ def set_cluster_config_overrides(self, cluster_id, config_values):
         return self.cloudv2._http_post(base_url=self.config.admin_api_url,
                                        endpoint='/SetClusterConfigOverrides',
                                        json=payload)
+
+    # Update cluster properties using Public API
+    def update_cluster_property_public(self, cluster_id, property_key,
```

Comment:
make this accept a map of Map[str, str]? then you'll be able to replace the _http_patch call from within the test with this. i.e. the one setting iceberg endpoint and such